pkg/target/target.go

```diff
@@ -352,7 +360,7 @@ func (h *K8sValidationTarget) MatchSchema() apiextensions.JSONSchemaProps {
 			},
 			"name": {
 				Type:        "string",
-				Description: "Name is the name of an object.  If defined, it will match against objects with the specified name.  Name also supports a prefix-based glob.  For example, `name: pod-*` would match both `pod-a` and `pod-b`.",
+				Description: "`name` is the name of an object.  If defined, it will match against objects with the specified name.  Name also supports a prefix-based glob.  For example, `name: pod-*` would match both `pod-a` and `pod-b`.",
```

Comment:
In general, try not to use future tense. "If defined, it matches against objects with the specified name." 

Like double-spacing after periods, I'm not going to call out all of these, but generally we don't use future tense."